### DISCUSSION TITLE
(docs) Walkthrough 2 incongruence in examples

### DISCUSSION THREAD

**Discussion by maxgitnet** [ago109/ngc-learn#6] [General]
Hello NAC people,

really enjoying this library.

In walkthrough 2 (docs), after model compilation has been showcased and when the settling routine is demonstrated, a node called "c" is clamped to a vector of ones with dim 1,10, and there is no node "c" defined in the walkthrough with dim=10, so that might stunt the flow if one is following along using the code examples.

Thanks

**Comment by ago109** [comment]
Thank you for catching this mistake in walkthrough 2 -- the issue has been corrected (as well the incorrect reference to `d` node as well). Originally, the 5-node circuit that one creates earlier was to be used for the compilation example, but the 3-node circuit was later created as a simpler/clearer example and the text/code snippets weren't completely updated to reflect the change.

Again, thanks for pointing this out -- the walkthrough 2 should now compile correctly and not refer to any absent/non-existent `c` (or even `d`) node now.

Really happy to hear you are enjoying the library and we really appreciate the error/bug catching!
